website/src/components/wordpress.tsx

```diff
@@ -226,19 +227,40 @@ const PullWords = (props: {
     levelOfDetail > LevelOfDetail.Pronunciation
       ? annotationSection.wordParts
       : annotationSection.story
+
+  let documentCitation = docContents.title + ", word " + props.first
+
+  if (props.last) {
+    documentCitation =
+      docContents.title + ", words " + props.first + " – " + props.last
+  }
+
+  let documentLink =
+    "https://dailp.northeastern.edu/documents/" +
```

Comment:
Will this link be changing once CWKW is launched? Is there an alternative to hard coding it?

Reply:
that GraphQL call would involve using CollectionChapter, right, since that's where the specific slug is stored? I am looking rn

Reply:
I've been looking for a bit-- I think this would be different since the url for CWKW is stored as the path in CollectionChapter and isn't necessarily the slug. I'm a bit confused on how to query CollectionChapter from document (?) or how the logic there would work to get that value. would it involve modifying an existing GraphQL query? Also, right now we only have one collection, CWKW, but as we expand, how will the specific collection name itself be stored? I couldn't find a value for that within CollectionChapter

Reply:
if i'm understanding your question correctly, yes i would avoid hard-coding this and instead utilize one of the defined routes in the file `routes.ts`. i think you should be able to use the `documentWordPath` route in that file but if the formatting is not the same, you can define a new route in that file to call here!

Reply:
I looked in the `routes.ts` file and wrote a new `collectionWordPath` route, but I'm still a little confused about what the best way to access the collection and chapter slug from the `wordpress.tsx` file would be